snuba/util.py

```diff
@@ -95,6 +96,16 @@ def function_expr(fn: str, args_expr: str='') -> str:
     to more complex expressions.
 
     """
+    if fn.startswith('apdex('):
```

Comment:
nit: we are trying to converge to double quotes for strings

Reply:
you mean for the python code itself? (not the column literal)

Reply:
yes, for the pyhton code itself